templates/CRM/Contribute/Form/Contribution/Main.tpl

```diff
@@ -91,7 +91,7 @@
   {elseif !empty($ccid)}
     {if $lineItem && $priceSetID && !$is_quick_config}
       <div class="header-dark">
-        {ts}Contribution Information{/ts}
+        {ts}Contribution Information{/ts} &ndash; {ts 1=$display_name}%1{/ts}
```

Comment:
Sorry, I should've noticed this earlier: your `{ts}` isn't wrapping any string to translate--just a variable to drop in as-is.  You could either:

- get rid of the `ts` and just drop it in as

  ```php
  {ts}Contribution Information{/ts} &ndash; {$display_name}
  ```

- or wrap the whole thing in a `ts` like

  ```php
  {ts 1=$display_name}Contribution Information &ndash; %1{/ts}
  ```

The latter causes more work for our translators (since it introduces a new string to translate), but it allows for localization of how the header is ordered and punctuated.  I don't know which is preferred as a general rule.

Reply:
(and this may depend upon how you address the other concern about whether the `$display_name` is populated)